README.md

```diff
@@ -28,20 +28,6 @@ Here's an example, using CFPB's regulation H.
 At the end, you will have new directories for `regulation`, `layer`,
 `diff`, and `notice` which would mirror the JSON files sent to the API.
 
-## Quick Start with Modified Documents
```

Comment:
I don't think I actually removed the contents, since this information was mostly repeated with the "Quick Start" section. I can bring that back, if the "Quick Start with Modified Documents" is better.